doc/cephfs/index.rst

```diff
@@ -47,7 +47,11 @@ For most deployments of Ceph, setting up your first CephFS file system is as sim
 The Ceph `Orchestrator`_  will automatically create and configure MDS for
 your file system if the back-end deployment technology supports it (see
 `Orchestrator deployment table`_). Otherwise, please `deploy MDS manually
-as needed`_. You can also `create other CephFS volumes`_.
+as needed`_. You can also `create other CephFS volumes`_. In case, data
+and/or metadata pools required for new FS already present, pass them as
+follows::
```

Comment:
If data or metadata pools for the new CephFS filesystem already exist, pass them as follows:

Reply:
This patch is present in this PR but will be removed eventually and in the original PR it has been removed.